src/vivarium_cluster_tools/psimulate/results/processing.py

```diff
@@ -108,11 +134,22 @@ def _concat_preserve_types(df_list: List[pd.DataFrame]) -> pd.DataFrame:
 
 @vct_utils.backoff_and_retry(backoff_seconds=30, num_retries=3, log_function=logger.warning)
 def _safe_write(df: pd.DataFrame, output_path: Path) -> None:
-    """Write a dataframe to disk, retrying on failure.
-
-    Writing to some file types over and over balloons the file size so instead we
-    write to a new file and move it over.
+    """Write a dataframe to disk, retrying if there are any issues.
+
+    Parameters
+    ----------
+    df
+        DataFrame to write to disk.
+    output_path
+        Path to write the DataFrame to.
+
+    Raises
+    ------
+    NotImplementedError
+        If the file extension is not supported.
     """
+    # Writing to some file types over and over balloons the file size so
+    # write to new file and move it over to avoid
```

Comment:
This was a problem with hdf, but is this still true with parquet? Can you investigate if it's easy or create a ticket to investigate it.